tests/unit/kvmagent/test_vm_plugin_handlers.py

```
@@ -1778,6 +1778,149 @@ def test_reload_redirect_usb(self):
+            '<domain><devices><shmem name="tf_vm-uuid"/></devices></domain>'
+        ))
+        vm.domain.detachDeviceFlags = MagicMock()
+        wait_callback_success = MagicMock(return_value=True)
+        rm_file_force = MagicMock()
+        monkeypatch.setattr(vm_plugin, 'get_vm_by_uuid_no_retry', MagicMock(return_value=vm))
+        monkeypatch.setattr(vm_plugin.linux, 'wait_callback_success', wait_callback_success)
+        monkeypatch.setattr(vm_plugin.linux, 'rm_file_force', rm_file_force)
+
+        req = _make_req({
+            'vmUuid': 'vm-uuid',
+            'shmem': {'path': '/dev/shm/tf_vm-uuid', 'size': 2 * 1024 * 1024},
```

Ruff (0.15.12)

[error] Probable insecure usage of temporary file or directory: "/dev/shm/tf_vm-uuid"

(S108)